### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by Iptysam**
Create a database named vet_clinic.
Create a table named animals with the following columns:
id: integer
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
Copy the SQL you used in the previous point into the schema.sql file in the generated repository.
Insert the following data:
Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
Copy the SQL you used in the previous point into the data.sql file in the generated repository.
Write queries for the following quests:
Find all animals whose name ends in "mon".
List the name of all animals born between 2016 and 2019.
List the name of all animals that are neutered and have less than 3 escape attempts.
List the date of birth of all animals named either "Agumon" or "Pikachu".
List name and escape attempts of animals that weigh more than 10.5kg
Find all animals that are neutered.
Find all animals not named Gabumon.
Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
Take a screenshot of the results of your queries.
Copy the SQL you used in the previous point into the queries.sql file in the generated repository.
date btn 2016 and 2019
ends mon
name not Gabumon
name escape
nurtered
nurtured and escape  less 3
weight btn 10 4 and17 3
dateofbirth either agumon or pikachu

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name like '%mon';
+SELECT * from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * from animals WHERE neutered AND escape_attempts <3;
+SELECT * from animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT (name, escape_attempts) from animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
+SELECT * from animals WHERE neutered;
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```
Comment (TuyishimireEric) [inline queries.sql:10]: - [ ] **Good job so far 👏 👏** However, it was suggested in the project requirements to add a screenshot of all query results to the pull request description. 😂 Kindly, add them as the result of all your queries 👍
 
image

image

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @Iptysam   👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight 
✔ correct SQL commands👍 
✔ Professional readme👍 

## Required Changes ♻️

- check the comments below
**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Iptysam 😊, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

Almost near

 ## Highlights
✔️ Animal table created properly
✔️ Answered analytical questions provided to you

 ## Required Changes ♻️

- [ ] As stated by the previous reviewer, great for adding the images in the repository under the screenshots folder but you need to add screenshots in the description of the pull request by clicking the following part as in the image below and choosing the image from your pc
>image

## Optional suggestions
- [OPTIONAL] It will  be better if your remove the broken logo from your readme or add your own one to make your readme professional

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Iptysam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**